pkg/apis/stream/v1alpha1/stream_validation.go

```diff
@@ -31,16 +31,20 @@ func (s *Stream) Validate(ctx context.Context) *apis.FieldError {
 	return errs
 }
 
-func (ss *StreamSpec) Validate(ctx context.Context) *apis.FieldError {
-	if equality.Semantic.DeepEqual(ss, &StreamSpec{}) {
+func (s *StreamSpec) Validate(ctx context.Context) *apis.FieldError {
```

Comment:
please revert

Reply:
it's reported by Goland. It suggests all receivers should be named consistently, I do not see the harm in that.

Reply:
Goland is wrong, the behavior was consistent before this change. `*Stream` != `*StreamSpec`

Reply:
yep, or the inconsistency is somewhere else, probably misread it